kivy/uix/image.py

```diff
@@ -168,7 +168,18 @@ def get_image_ratio(self):
     defaults to False.
     '''
 
-    anim_delay = NumericProperty(.25)
+    durations = ListProperty(None, allownone=True)
+    '''Delay the animation if the image is sequenced (like an animated gif).
+    Set if frame delay is variable, not constant.
+    Ignored if anim_delay is set to -1 or auto_anim_delay set to True.
```

Comment:
Isn't it the opposite that `auto_anim_delay` is ignored if `durations` is specified?

So if I understand it correctly, this should say: 
```
"Ignored if :attr:`anim_delay` is set to ``-1``. If :attr:`durations` is specified, :attr:`auto_anim_delay` will be ignored and treated as if it's False."
```

Unless you meant like you wrote it, and `auto_anim_delay` is dominant? But then above `            self._auto_anim_delay = False` should probably be deleted!? Given that we have `auto_anim_delay` it makes sense to me that it should be dominant, even when `durations` is specified. Because otherwise what's the point of having it? 

From your implementation I'm thinking that maybe you'd prefer `durations` to be dominant if specified. But if not specified we look at `auto_anim_delay` to decide whether to use the constant delay or the file provided delays. Is that right? 

A third alternative is for `auto_anim_delay` to be dominant, but fallback on `durations` or `anim_delay` when it's True but the file loader doesn't provide delays for the images. I think this version makes the most sense to me, but we should decide which you like and make sure the docs are consistent with it :)